### PR TITLE
fix: snowtrace is online but not all contracts are verified

### PR DESCRIPTION
**Pull request by gosuto-inzasheru**
issue is that some of our contracts are not verified on snowtrace, eg https://snowtrace.io/token/0x72dD9b41FEc59cf58140cAb9C92CEfC8F212354D/contract/code?chainId=43114

made that more explicit in this fix. also if verified on snowtrace the report should pass instead of skip avax altogether

### CODE REVIEW

**Comment by gosuto-inzasheru** [comment]
on hold, i can fix this completely using snowscan.xyz. wip

**Comment by gosuto-inzasheru** [comment]
moving to snowscan instead

File: action-scripts/brownie/scripts/report_gauges.py [hunk 1]
```diff
@@ -75,23 +75,11 @@ def _extract_pool(
...
-            pool_id = "AVAX_UNKNOWN"
-            pool_address = "AVAX_UNKNOWN"
-            a_factor = "AVAX_UNKNOWN"
-            fee = "AVAX_UNKNOWN"
-            tokens = []
-            rate_providers = []
-        else:
-            network_id = ADDR_BOOK.chain_ids_by_name[chain.replace("-main", "")]
-            switch_chain_if_needed(network_id=network_id)
+        network_id = ADDR_BOOK.chain_ids_by_name[
```
Comment (Xeonus) [inline action-scripts/brownie/scripts/report_gauges.py:78]: Thanks for the clean-up!

File: action-scripts/brownie/networks.yaml [hunk 2]
```diff
@@ -10,7 +10,7 @@ live:
   - name: Avalanche
     networks:
       - chainid: 43114
-        explorer: https://api.routescan.io/v2/network/mainnet/evm/43114/etherscan/api
+        explorer: https://api.snowscan.xyz/api
```
Comment (Xeonus) [inline action-scripts/brownie/networks.yaml:13]: As per Etherscan team, this is the official deployment for AVAX now

**Review by Xeonus — APPROVED** [review]
Looks good:
- adding new API endpoint
- Cleanup of snowtrace legacy "temp fixes"

**Pull request merged**